smashing.html

```diff
@@ -3,26 +3,11 @@
     <head>
         <title>HTML page - Large</title>
         <meta charset="utf-8">
-        <style type="text/css">
-            body{background: #ffffff; margin: 0px; font-size: 14px;}
-            a:link{color:#41b7d8;}
-            a:visited{color:#41b7d8;}
-            a:hover{color:blue;}
-            figure{margin: 0px; display: inline;}figure{margin: 0px; display: inline;}
-            header{background: #eeeeee; padding: 15px;}
-            article{padding: 15px; font-size: 1.25em;}
-            article h1 link:a{color:#000000 !important;}
-            article figure{margin: auto; display: block;}
-            h1{font-size: 2em;}
-            h2{font-size: 1.5em;}
-            h3{font-size: 1.25em;}
-            nav#paging-bottom a{padding: 15px; font-size: 1.25em;}
-            footer{padding: 15px; font-size: 1em;}            
-        </style>
+        <link rel="stylesheet" href="css/styles.css">
     </head>
     <body>
         <header>
-            <figure><a href=""><img src="https://media-mediatemple.netdna-ssl.com/wp-content/themes/smashing-magazine/assets/images/logo.svg" width="143"/></a></figure>
+            <figure><a href=""><img src="img/logo.svg" width="156"/></a></figure>
```

Comment:
This can be spread into lines:
```
<figure>
    <a href="">
        <img src="img/logo.svg" width="156"/>
    </a>
</figure>
```

Also, `<figure>` is not right here, because it should only exist inside an article so it means that it's a reference of the article. I mean, not every image on the site should be a figure.